lib/cucumber/core/test/runner.rb

```diff
@@ -4,40 +4,39 @@ module Cucumber
   module Core
     module Test
       class Runner
-        attr_reader :report, :running_test_case, :running_test_step
-        private :report, :running_test_case, :running_test_step
+        attr_reader :events, :running_test_case, :running_test_step
+        private :events, :running_test_case, :running_test_step
 
-        def initialize(report)
-          @report = report
+        def initialize(events)
+          @events = events
         end
 
         def test_case(test_case, &descend)
```

Comment:
I do think we could remove some duplication here between the `test_case` and `test_step` methods, similar to what I suggested in a couple other places.